### PR TITLE
✨ Support for Apple Silicon (arm64) Macs

### PR DESCRIPTION
**Pull request by chris-araman**
Updating dependencies to pick up support for Apple Silicon (arm64).

Relevant PRs:
https://github.com/Quick/Nimble/pull/827
https://github.com/xcconfigs/xcconfigs/pull/98

This will aid in enabling arm64 support for https://github.com/mas-cli/mas/issues/308.

FYI @phatblat @fabriceb

### CODE REVIEW

File: .travis.yml
Comment (elliottwilliams) [inline .travis.yml:46]: Does this PR drop support for Swift 5.0 and/or Xcode 10.2's sdk? If so, that's probably something we should be cautious about.

Comment (chris-araman) [reply .travis.yml:46]: Yes, I believe it would drop support for Swift 5.0 because Quick 3.0.0 and Nimble 9.0.0 require Swift 5.2 and Xcode 11.4.

Alternatively, we could urge the maintainers of Quick and Nimble to port Apple Silicon support back to Quick 2.x and Nimble 8.x.

Incidentally, I think we'll still need to wait for a subsequent release of Quick, as https://github.com/Quick/Quick/commit/1172a5df7ee3da5817cf7c9771c10aeb93b770f4 wasn't in the 3.0.0 release.

**Comment by fabriceb** [comment]
Thanks for that @chris-araman !

File: .travis.yml
Comment (elliottwilliams) [reply .travis.yml:46]: Got it, thanks!

**Comment by rgoldberg** [comment]
Does anything else need to be done before this can be merged?

**Comment by chris-araman** [comment]
I think we still need Quick 3.0.1 to be released with https://github.com/Quick/Quick/commit/1172a5df7ee3da5817cf7c9771c10aeb93b770f4 from @ikesyo.

**Comment by rgoldberg** [comment]
That commit was merged into master 4 months ago.

Why would they wait so long to release something that makes their software work on Apple Silicon?

**Comment by chris-araman** [comment]
@elliottwilliams, I think we have everything we need here now that @ikesyo has released Quick 3.0.1. Removing the `[WIP]` tag. Ready for your review.

**Comment by chris-araman** [comment]
I've picked up @ikesyo's fix for arm64 with Quick 3.1.0. (Thanks, @ikesyo!)

@elliottwilliams, this is now ready for your review and release.

File: .travis.yml
Comment (phatblat) [reply .travis.yml:46]: @ikesyo I would love to hear your quick thoughts about these Swift version changes as you've been handling that well for several of the mentioned projects.

**Review by phatblat — APPROVED** [review]
Thanks @chris-araman!

**Pull request merged**
